src/public/Convert-SentinelARArmToYaml.ps1

```diff
@@ -44,63 +75,76 @@ Get-Content -Path "C:\Temp\MyRule.json" -Raw | Convert-SentinelARArmToYaml -OutF
 function Convert-SentinelARArmToYaml {
     [CmdletBinding(DefaultParameterSetName = 'StdOut')]
     param (
-        [Parameter(Mandatory = $true,
+        [Parameter(Mandatory,
```

Comment:
There is a bug when using `-Force` and `UseDisplayNameAsFilename` and `UseIdAsFilename` from StdOut.
The parameter `Filename` must not be mandatory. Fix see below

```powershell
        [Parameter(Mandatory,
            Position = 0,
            ParameterSetName = 'Path')]
        [Parameter(Mandatory = $false,
            Position = 0,
            ParameterSetName = 'UseOriginalFilename')]
        [Parameter(Mandatory = $false,
            Position = 0,
            ParameterSetName = 'UseDisplayNameAsFilename')]
        [Parameter(Mandatory = $false,
            Position = 0,
            ParameterSetName = 'UseIdAsFilename')]
        [Parameter(Mandatory,
            Position = 0,
            ParameterSetName = 'StdOut')]
        [string]$Filename,
```

Reply:
I may be incorrect but just to check, I thought we may also want this to work:
```powershell
get-content /workspaces/SentinelARConverter/tests/examples/ScheduledMultiple.json -raw | Convert-SentinelARArmToYaml
```
This implies we have the `Pipeline` and `StdOut` Parameterset? At the moment this doesn't out anything, the same applies with a single ART:
```powershell
get-content /workspaces/SentinelARConverter/tests/examples/Scheduled.json -raw | Convert-SentinelARArmToYaml        
```